<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Alogorithm Trading</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>설정</string>
    </property>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>거래시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>거래중지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReStart">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>거래재시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lePrice">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>투입금</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>계좌정보</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌번호</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbAcctNo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>예수금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leD2Price">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>거래</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>조건검색명</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCdtNm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>총투입금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTotalPrice">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>수익금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leProfits">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>수익률</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leProfitsPc">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>거래정보</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매수종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>매도종목</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteBuyLog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>101</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteSellLog">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>101</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>361</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>내역</string>
    </property>
    <widget class="QPlainTextEdit" name="pteLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
